recipes/pyhocon/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install 
```

Comment:
Please remove this file and bld.bat, and instead add the script entry in meta.yaml (I'll add an example below)